mappings/net/minecraft/unused/UnknownWorldConstants6148.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_6148 net/minecraft/unused/UnknownWorldConstants6148
```

Comment:
While it's nice to reach 100% mapped classes, I think it makes no sense to name a class like that. Better to live unmapped

Reply:
No? According to the reasoning above, all package info should be unmapped. This mapping at least have some information about what this class is; more info than an unmapped class. At least we know this is some world-related constant from this name; it's definitely more informative than `class_6148`.

Once before, we had a "sortme" package https://github.com/FabricMC/yarn/tree/18w50a/mappings/net/minecraft/sortme. This mapping is generally the same idea; we should make use of what information we currently have.

Reply:
Because you state that this is world-related based on the package structure, and we can't be sure of that. The constant values are way too broad to be able to guess anything.
I remember someone saying on discord that it could also be related to redstone-related block power, but these values probably exist in entity classes, item classes or any other game part.

What I mean is naming this class like you did, with the docs you wrote, will make people think that this is 100% sure world-related.

Reply:
And as well, naming a class for the sake of deobfuscating a class doesn't make sense. It makes sense for package info classes because we don't need them. But for this class, putting them in a package with "unknown" name will just burry it and we won't look at it anymore.
This is not a "sortme" or "temp" package, this is "unused". This is wrong. These constants are used by Mojang, like other constant classes. We just can't see it.

Reply:
Done. Now this class is unnamed while its javadocs are kept.

Reply:
in official mappings this constants used for redstone signal:
![image](https://user-images.githubusercontent.com/18630111/118393771-6c410380-b649-11eb-973f-bd5b3cebd07c.png)

Reply:
@andreypfau Yarn naming cannot use Mojang's official mapping data for legal reasons (see CONTRIBUTING.md). Mojmap names can't be shared in Yarn PRs/issues either since they risk "tainting" the contributors with Mojmap names.